### ISSUE TITLE
Question or Sugestion

### ISSUE THREAD

**Issue by avdoviic** [TheAlphamerc/flutter_twitter_clone#11]
How to make that feed page refreshing changes fast, bc when I chang for example my profile picture or username , on feed page stay still the old settings or pictures..

**Comment by avdoviic** [comment]
Also the feed page don't showing the post for the folower till not unfollow und follow again same person..

**Comment by TheAlphamerc** [comment]
If you navigating to someone's profile and follow him, his tweets will immediately started to show on your home page.

**Comment by avdoviic** [comment]
> If you navigating to someone's profile and follow him, his tweets will immediately started to show on your home page.

Yes this work for some time, but after some  closing app und reopening  the post don't showing anymore, und after unfollowing und again following this show again.. Und like this repeating, work for some time und after this  stoping to work und again unfollow und follow und this start  to work again..

**Comment by TheAlphamerc** [comment]
Please let me know which version of `Fwitter` you are using?

**Comment by avdoviic** [comment]
> Please let me know which version of `Fwitter` you are using?

Last update, but tis is not big broblem, the one issues what I'm finded is that when I tap fast 7-8 time follow or unfollow button for some profile, the app changing my account with this profile account  und you can easy post what you want on this profile or to change what you want, und after restarting app the changing again on my  real account.. This not happening too ofen, one time in 5 attempts but I think is not good, sorry for the comments but I want to help, to tell for the things what I finding when something is not okay.. :)

**Comment by TheAlphamerc** [comment]
Thanks @avdoviic to take interest in `Fwitter` project and letting me know about bug, I will try to resolve this in the next update.

**Comment by avdoviic** [comment]
Your welcome, one more suggestion is that it will be very nice if coments are not showing on the feed page, bc when you have more coments on some post you will need to watch this even if you don't want. :)

But Project is the best till now off all clones what I know..  👍 👍 👍 👍 👍 👍 🥇

**Comment by TheAlphamerc** [comment]
@avdoviic your suggestion makes sense. Comments appear in large numbers and definitely user didn't want to see every comment on his timeline but on the other hand if we check `Twitter`, every comment is an independent tweet itself so being a clone `Fwitter` should behave like Twitter.

**Comment by avdoviic** [comment]
Yes you are right, but I think that Twitter don't showing all comments on the feed page, that showing 2-3 comments maximum und the other you can seen when you open post, bc when you have for example 100 coments, you can't seen nothing else for other people.. und I think this will be good to implement.  Thank you for answering.. 👍

**Comment by avdoviic** [comment]
How to add more different icon in the "constant.dart" , I can't find galleries with hash code icons, und I need some for "logout" und some other also..?
